### ISSUE TITLE
Поддержка списков с пометками

### ISSUE THREAD

**Issue by shureek** [Jenyay/outwiker#368]
Хотелось бы, чтобы markdown поддерживал такие вот списки с пометками:
```
- [ ] Раз
- [ ] Два
- [ ] Три
```
- [ ] Раз
- [ ] Два
- [ ] Три

**Comment by adeptg** [comment]
Да, было бы здорово. Особенно, если было бы можно им ставить галочки и сохранять состояние. И еще совсем здорово - опция зачеркивать пункт если стоит галочка (примерно как это делает google keep).
P.S. Да, я пытаюсь делать check lists :)
